src/js/exporters.js

```diff
@@ -7,18 +7,12 @@ function export_error() {
   }
 }
 
-function save_file(filename, contents, raw) {
-  var link = document.createElement('a');
-  link.download = filename;
-
-  if (raw === true) {
-    const blob = new Blob([contents], { type: 'application/octet-stream' })
-    link.href = window.URL.createObjectURL(blob)
-  } else {
-    link.href = "data:application/octet-stream," + encodeURIComponent(contents);
-  }
-
-  link.dispatchEvent(new MouseEvent(`click`, { bubbles: true, cancelable: true, view: window })); // opens save dialog
+function save_file(filename, contents, mimeType = 'application/octet-stream,') {
```

Comment:
I think I missed the change from Azorlough and ignorantly copied this over from my old develop branch. I put the 'raw' arg back in the same spot and moved the 'mimeType' arg up one spot. I'm not entirely sure this is an ideal definition since the 'mimeType' arg won't matter if 'raw' is true - I am not sure if there are situations where you'd want to use a blob but change the mimeType. But this change makes the FL exports work again.